src/backend/.gitignore

```diff
@@ -0,0 +1,34 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**
+!**/src/test/**
+
+### STS ###
```

Comment:
Out of curiosity, what's STS?  the .project and .classpath are all that Eclipse generates, but these are already excluded from the parent .gitignore on the repo root

Reply:
Not sure actually. It is a generated file I took from the efiling project. It might be spring initilizr generating it.